CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 # Change Log
 
+## UNRELEASED
+
+- ci/fix: improve error handling in release-info script [#815](https://github.com/hypermodeinc/modus/pull/815)
+
```

Comment:
in this case, no need for changelog entry because it's not associated with a release and goes into effect right away.